backend/services/test-plan/src/main/java/io/metersphere/plan/service/TestPlanReportService.java

```diff
@@ -1434,4 +1475,19 @@ public List<String> getActualReportIds(String reportId) {
             return List.of(reportId);
         }
     }
+
+    /**
+     * 获取报告汇总详情
+     * @param reportId 报告ID
+     * @return 汇总详情
+     */
+    private TestPlanReportSummary getReportSummary(String reportId) {
+        TestPlanReportSummaryExample example = new TestPlanReportSummaryExample();
+        example.createCriteria().andTestPlanReportIdEqualTo(reportId);
+        List<TestPlanReportSummary> testPlanReportSummaries = testPlanReportSummaryMapper.selectByExampleWithBLOBs(example);
+        if (CollectionUtils.isEmpty(testPlanReportSummaries)) {
+            throw new MSException(Translator.get("test_plan_report_detail_not_exist"));
+        }
+        return testPlanReportSummaries.getFirst();
+    }
 }
```

Comment:
The code seems to be a part of the "TestPlanReport" package and it appears that there is an error where multiple methods related to different functionalities overlap.

For instance, one method `selectById(String reportId)` returns a response object `TestPlanReport`. Another method `calcTaskExecActual` takes two parameters: `String reportId`, which refers to the specific report being processed (`TestPlanReport`). However, inside this method, another function call `checkReport` is attempted, which could have been used for some other purpose earlier in the source code but seems unnecessary now because no further operations were found around it.

Therefore, I recommend removing the redundant calls or finding what was originally intended with these functions before merging them back into the core of your project. If you need any more clarifications about your question, feel free to ask!